### DISCUSSION TITLE
CrimeFlare doesn't work

### DISCUSSION THREAD

**Discussion by evgenycc** [zidansec/CloudPeler#4] [General]
Hello. The application is currently not working. It has been unavailable since yesterday. Tried another app in your repository on GitHub, but it too is in "Application error" status. On heroku.com

**Comment by zidansec** [comment]
> Hello. The application is currently not working. It has been unavailable since yesterday. Tried another app in your repository on GitHub, but it too is in "Application error" status. On heroku.com

Yes, the hosting where the crimeflare API is deployed has expired.

**Reply by emyfops** [reply]
Could you possibly host it somewhere else ? Like a VPS

**Reply by zidansec** [reply]
Yes, I'm short on funds for that.

**Reply by emyfops** [reply]
https://www.hostinger.com/

**Comment by zidansec** [comment]
CrimeFalare is now reusable

**Reply by evgenycc** [reply]
That's great! Thank you so much for your work!

**Comment by evgenycc** [comment]
Hi. Sorry for the perhaps incorrect question: "How does your Cloudflare address discovery project work?" 
I'm at a loss for ideas. And so I want to ask a couple of questions:
1. Is a database used to look up addresses? (After all, the search works too fast for any kind of server scanning);
2. If you use an address database, how do you populate it? (How are addresses searched for that database? Or does it not fill up, but is it static, i.e. does it not change over time?)

Once again, I apologize for, perhaps, incorrect questions.

Respectfully, Johan

**Comment by evgenycc** [comment]
Why I ask these questions. It's simple. It turns out that we are too dependent on the availability of hosting and other factors in your project. And if something goes wrong, a lot of projects that will be based on yours will simply stop working. So, I want to understand how it works. Or at least get a direction in which to search.

Thanks

**Comment by zidansec** [comment]
This tool is back in operation, but need to know. This tool is not 100% accurate, because as I explained. The way this tool works takes advantage of Cloudflare's leaked database in 2018. New API [https://crimeflare.zidansec.com](https://crimeflare.zidansec.com)
